README.md

```diff
@@ -0,0 +1,44 @@
+# Picknik UR5 Moveit Config
+
+UR5 launch files, calibration data, and urdf files to simplify using the UR5.
+
+Developed by Mike Lautman and Tyler Weaver at [PickNik Consulting](http://picknik.ai/)
+
+## Feature Status
+
+Currently Supported:
+
+* Table collision object
+* Calibration from UR5
+* Simulation with Rviz
+* Fake controller to test without hardware
+* OEM UR Driver
+
+Roadmap:
+
+* UR modern driver
+* Fully document arguments to launch files
+
+## Example Use
+
+The [ur5_examples](https://github.com/PickNikRobotics/ur5_examples) repo has examples for using these launch files.
+
+## demo.launch
+This is the primary launch file that uses other launch files to start the example.
+
+### Simulation rosgraph (fake_execution:=true)
```

Comment:
No need for a rosarg here, this is already the default.

Also, I'm not a fan of documenting roslaunch args in the README. Better to keep documentation in the roslaunch file itself and instruct the user to use `--ros-args` to learn more about the available arguments, otherwise documentation gets out of sync.